packages/grid/src/styled/StyledCol.ts

```diff
@@ -55,44 +58,31 @@ const retrieveColClassNames = ({
   return output;
 };
 
-/**
- * Accepts all `<div>` props
- */
-const Col = styled.div.attrs(props => ({
+export interface IStyledColProps {
+  size?: number | string;
+  xs?: number | string | boolean;
+  sm?: number | string | boolean;
+  md?: number | string | boolean;
+  lg?: number | string | boolean;
+  xl?: number | string | boolean;
+  offsetXs?: number | string | boolean;
+  offsetSm?: number | string | boolean;
+  offsetMd?: number | string | boolean;
+  offsetLg?: number | string | boolean;
+  offsetXl?: number | string | boolean;
+  alignSelf?: 'start' | 'center' | 'end';
+  justifyContent?: 'start' | 'center' | 'end' | 'around' | 'between';
+  order?: any;
```

Comment:
I noticed that `StyledCol` component takes order as a string [here](https://github.com/zendeskgarden/react-components/pull/528/files#diff-e9d9dd98622fd439d2d246cb9d144f60R143). Would this be better suited to be typed as `string`? Or does it also accept other types?

Reply:
I'm cleaning this up in the CSS-in-JS PR. Stay tuned ;)